cmd/event-driver-sidecar/main.go

```diff
@@ -0,0 +1,22 @@
+///////////////////////////////////////////////////////////////////////
+// Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+///////////////////////////////////////////////////////////////////////
+
+package main
+
+// NO TEST
+
+import (
+	"os"
+
+	"github.com/vmware/dispatch/pkg/event-sidecar"
+)
+
+func main() {
+	cli := eventsidecar.NewCmd(os.Stdin, os.Stdout, os.Stderr)
+	if err := cli.Execute(); err != nil {
+		os.Exit(1)
```

Comment:
Don't you want to dump err before exiting?

Reply:
The idea here was that error is only used to indicate the return code, and the actual error message is printed by the command. But then I realized I use `log.Fatal` in command so it won't even reach this section... I will rewrite it ;)